coordinape/coordinape
# Allow Deleted Users to process Claims

# Motivation

Deleted Users do not have access to old epoch data. This refactor allows
us to render claims from users who have been deleted and allows users to
process those claims in our frontend.

Circle and Org names have been replace with 'Uknown' but we could
just leave them blank

# Test Plan

Existing unit and e2e tests still pass.

The rest of the testing is manual:
1. Fund a vault and create some distributions
2. delete a user who received funds from those distributions
3. log into the frontend as that user and attempt to claim the funds
4. verify that the claim is successful and the claim row moves to the
   history table
